zyte_common_items/items.py

```diff
@@ -1136,3 +1139,46 @@ class JobPosting(Item):
     metadata: Optional[JobPostingMetadata] = attrs.field(
         default=None, converter=attrs.converters.optional(MetadataCaster(JobPostingMetadata)), kw_only=True  # type: ignore
     )
+
+
+@attrs.define(kw_only=True)
+class SocialMediaPost(Item):
+    """Represents a single social media post."""
+
+    #: The url of the final response, after any redirects.
+    url: str = attrs.field(converter=url_to_str)
+
+    #: The identifier of the post.
+    postId: Optional[str] = None
+
+    #: Details of reactions to the post.
+    reactions: Optional[Reactions] = None
+
+    #: The text content of the post.
+    text: Optional[str] = None
+
+    #: The title of the post.
+    title: Optional[str] = None
+
+    #: The description of the post.
+    description: Optional[str] = None
+
+    #: The timestamp at which the post was created.
+    #:
+    #: Format: Timezone: UTC. ISO 8601 format: "YYYY-MM-DDThh:mm:ssZ"
+    datePublished: Optional[str] = None
+
+    #: The list of hashtags contained in the post.
+    hashtags: Optional[List[str]] = None
+
+    #: The list of URLs of media files (images, videos, etc.) linked to the post.
```

Comment:
It seems copied from upstream, but could it be a typo?

```suggestion
    #: The list of URLs of media files (images, videos, etc.) linked from the post.
```